mama_pipeline.py

```diff
@@ -549,7 +588,7 @@ def mama_pipeline(sumstats: Dict[PopulationId, Any], ldscore_list: List[Any], sn
         sumstats[(ancestry, phenotype)] = new_df
 
         # Report mean chi squared
-        mean_chi_2 = np.square(new_df[Z_COL].to_numpy()).mean()
+        mean_chi_2 = calculate_mean_chi_sq(new_df[Z_COL].to_numpy())
         logging.info("\t\tMean Chi^2 for %s = %s", (ancestry, phenotype), mean_chi_2)
```

Comment:
In the PR description, it seems like you wanted the mean chi2 printed on debug. Just wanted to flag that as it seems like it'll log at a level up.

Reply:
Also -- is this before or after QC? Do you think it would be helpful to include that in the log?

Reply:
Oh, this is the mean chi squared for the outputs of MAMA.  My understanding is that we'll want these all the time by default.  I suppose the text could be changed to indicate that this is for outputs, though it is in the output section of the log, so perhaps that would be redundant.  This is definitely after QC, btw, since it's also after running MAMA on the inputs.

Reply:
Ah gotcha ok that makes much more sense.